AutoRest/Generators/CSharp/Azure.CSharp.Tests/AutoRest.Generator.Azure.CSharp.Tests.csproj

```diff
@@ -16,6 +16,9 @@
     <AutoRestTest>true</AutoRestTest>
     <NuGetPackageImportStamp>2c7e2ccf</NuGetPackageImportStamp>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Net45-Debug'">
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
```

Comment:
to avoid the code duplicate.
I suggest we do it below https://github.com/Azure/autorest/blob/dev/tools/AutoRest.Settings.targets#L33
<CodeAnalysisRuleSet Condition=" $(AutoRestTest) == 'true' ">MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>